src/api/jest/index.ts

```diff
@@ -0,0 +1,27 @@
+import { Config } from '@jest/types';
+
+import jestPreset from '../../../jest-preset';
+import { mergeRaw } from '../../cli/configure/processing/record';
+
+type Props = Pick<
+  Config.InitialOptions,
+  | 'collectCoverage'
+  | 'collectCoverageFrom'
+  | 'collectCoverageOnlyFrom'
+  | 'coveragePathIgnorePatterns'
+  | 'coverageThreshold'
+  | 'globals'
+  | 'globalSetup'
+  | 'globalTeardown'
+  | 'setupFiles'
+  | 'setupFilesAfterEnv'
+  | 'testPathIgnorePatterns'
+>;
```

Comment:
@72636c Sorry to comment on an old PR, but I'm trying to understand why only a subset of jest configuration options is "Pick"ed in this `type Props`. Couldn't teams be given access to all Jest options? Ones that we would like to use for instance is https://jestjs.io/docs/configuration#clearmocks-boolean and https://jestjs.io/docs/configuration#resetmocks-boolean 

I can definitely add those as types to the list or ignore the typescript issue in our project, just wondering the background context as to why its limited :)

Reply:
Hey @benjaminpearson! The idea was to only expose options that "safely" extend our preset and wouldn't conflict with things we'd want to manage internally now or in future. But given how long this allowlist has grown, I'm 50:50 on whether to keep extending it or just allow the full list through.

Those two look perfectly reasonable (I wonder if they could even be defaults at some point).

Reply:
This should be resolved in the next release: #798